clientv3/README.md

```diff
@@ -72,6 +72,14 @@ if err != nil {
 }
 ```
 
+## Metrics
+
+etcd client provides metrics reporting. This can enable the operators to know how clients are performing and which client generates the most load.
```

Comment:
If someone doesn't know what metrics are in general, I don't think the etcd docs are the place for learning about it.